_posts/2017-06-28-Openshift-an-introduction.md

```diff
@@ -0,0 +1,241 @@
+It integrates with Red Hat Enterprise Linux 6 and is tested via Red Hat's QA process in order to offer a stable, supportable product with may be important for enterprises.
+
+### Openshift dedicated
+Openshift dedicated is the latest offering of openshift.
+It's OpenShift 3 hosted on AWS and maintained by Red Hat but it is dedicated to you
+
+### Openshift online
+OpenShift Online is managed by Red Hat’s OpenShift operations team, and quickstart templates enable developers to push code with one click, helping to avoid the intricacies of application provisioning.
+You can view it as openshift delivered as a SAAS(software as a service)
+
+# Benefits for developers
+Before I show you how easy openshift is for a developer, let me quickly explain Source to image (S2I) to make life even more easier.
+
+
+Let's show how easy your life can be with the following image
+
+> __source to image__ is a toolkit and workflow that creates a deployable docker image based on your source code and add it to the image registry. You don't even need a docker file anymore.It combines source code with a corresponding builder image from the integrated docker registry
+
+So now you know S2I let's take a look at the next picture
+<p>
+  <img class="image fit" alt="devopsoverview" src="/img/openshift-an introduction/devopsoverview.jpg">
+</p>
+* __code:__
+If your a developer I assume you know how to code and push it to git, so nothing new here ..
+
+* __Build:__
+The developer can push code to be built and run on OpenShift through their software version control solution or OpenShift can be integrated with a developer's own automated build and continuous integration/continuous deployment system. Here is were S2I can get usefull.
+
+* __deploy:__
+OpenShift then orchestrates where application containers will run and manages the the application to ensure it's available for end users.
+
+* __manage:__
+With your app running in the cloud you can monitor, debug, and tune on the fly.
+Scale your application automatically or allocate capacity ahead of time
+
+
+# A deeper look 
+Time to get a little bit more technical and take a deeper look how it works.
+I already talked about the developer part of the picture below, so let's focus on the rest!
+
+<p>
+  <img class="image fit" alt="deeperlook" src="/img/openshift-an introduction/deeperlook.svg">
+</p>
+
+
+### Infrastructure
+Openshift runs on your choice of infrastructure (Physical, Virtual,Private, Public)
+
+### Nodes
+A node provides the runtime environments for containers.
+Each node in a Kubernetes cluster has the required services to be managed by the master. 
+OpenShift creates nodes from a cloud provider, physical systems, or virtual systems.
+Kubernetes interacts with node objects that are a representation of those nodes.
+A node is ignored until it passes the health checks, and the master continues checking nodes until they are valid.
+In Openshift nodes are instances of RHEL(Redhat enterprise linux)
```

Comment:
In OpenShift, nodes ...
RHEL (Redhat Enterprise Linux) (capitalize)
(add space after "RHEL")